web/templates/settings/generate_integration_url_config_checkbox_modal.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="input-group" id="integration-url-{{key}}-container">
+    <label class="checkbox">
+        <input type="checkbox" id="{{key}}" class="integration-url-parameter" />
```

Comment:
Please don't use the raw key as an ID; that can do nasty things if it collides with IDs of other elements in the app. I think `integration-url-{{key}}-checkbox` is probably fine.